data-access-analysis-tools/adql-recipes.rst

```diff
@@ -0,0 +1,222 @@
+For this use-case, it is recommended to use the ``SELECT TOP`` statement, like in the example below that just retrieves the first 100 rows of the ``Object`` table.
+
+.. code-block:: SQL
+    SELECT TOP 100 * FROM dp02_dc2_catalogs.Object
+
+
+
+.. _Adql-Recipes-Cone-Search:
+
+Cone Search
+===========
+
+Retrieve the ``coord_dec`` and ``coord_ra`` columns from the ``Object`` table for objects within a 0.05 degree radius of RA = 62, Dec = -37.
+
+.. code-block:: SQL
+
+   SELECT coord_dec, coord_ra 
+   FROM dp02_dc2_catalogs.Object 
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), 
+   CIRCLE('ICRS', 62, -37, 0.05)) = 1
+
+
+
+.. _Adql-Recipes-FluxToMags:
+
+Convert fluxes to magnitudes
+============================
+
+As above, retrieve the ``coord_dec`` and ``coord_ra`` columns from the ``Object`` table for objects within a 0.05 degree radius of RA = 62, Dec = -37,
+and also retrieve the g-band AB magnitude and magnitude error.
+The ``scisql`` functions used below can be applied to any flux column.
+
+.. code-block:: SQL
+
+   SELECT coord_dec, coord_ra, 
+   scisql_nanojanskyToAbMag(g_calibFlux) AS g_calibMag, 
+   scisql_nanojanskyToAbMagSigma(g_calibFlux, g_calibFluxErr) as g_calibMagErr 
+   FROM dp02_dc2_catalogs.Object 
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), 
+   CIRCLE('ICRS', 62, -37, 0.05)) = 1
+
+
+.. _Adql-Recipes-Table-Joins:
+
+Table joins
+===========
+
+Below, the Source and CcdVisit table are joined in order to obtain the date and seeing from the CcdVisit table.
+Any two tables can be joined so long as they have an index in common.
+
+This query also renames (nicknames) columns and tables using ``AS``,
+and applies a spatial constraint, a temporal constraint (using ``obsStartMJD``), 
+and constraints on the band, extendedness, and flux value.
+
+Additional external resources on SQL table joins:
+ - `W2 School's SQL tutorial: joins <https://www.w3schools.com/sql/sql_join.asp>`__
+ - `The Data School: SQL Joins Explained Visually <https://dataschool.com/how-to-teach-people-sql/sql-join-types-explained-visually/>`__
+
+.. code-block:: SQL
+
+   SELECT src.ccdVisitId AS src_ccdVisitId, 
+   src.extendedness AS src_extendedness, 
+   src.band AS src_band, 
+   scisql_nanojanskyToAbMag(src.psfFlux) AS src_psfAbMag, 
+   cv.obsStartMJD AS cv_obsStartMJD, 
+   cv.seeing AS cv_seeing 
+   FROM dp02_dc2_catalogs.Source AS src 
+   JOIN dp02_dc2_catalogs.CcdVisit AS cv 
+   ON src.ccdVisitId = cv.ccdVisitId 
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), 
+   CIRCLE('ICRS', 62.0, -37, 1)) = 1 
+   AND src.band = 'i' 
+   AND src.extendedness = 0 
+   AND src.psfFlux > 10000 
+   AND cv.obsStartMJD > 60925 
+   AND cv.obsStartMJD < 60955
+
+
+
+.. _Adql-Recipes-Truth-Summary:
+
+TruthSummary and MatchesTruth table joins
+=========================================
+
+The query below demonstrates how to retrieve the truth table identifier (``id_truth_type`` from the ``MatchesTruth`` table)
```

Comment:
Do people know what truth table means at this stage of reading the documentation? I didn't see any explanation earlier in the repo but it could be on an earlier page I didn't read. If its self explanatory, please disregard suggestion.

Reply:
Perhaps not but, none of the other tables are defined on this page either. It's not self-explanatory, but there is already, up at the top, a relevant link to the DP0.2 data products definitions (_"Learn more about the [TAP-accessible DP0.2 catalogs](https://dp0-2.lsst.io/data-products-dp0-2/index.html#catalogs)."_). So I think that's sufficient.